### DISCUSSION TITLE
过期：懒人包 2023V1

### DISCUSSION THREAD

**Discussion by hooke007** [hooke007/mpv_PlayKit#251] [回收]
## 版本专帖
0107 弄错了rifecuda的依赖，，但是它居然能正常运行...重传以修复，如果你没有遇到问题，就不用管。如果需要更新，先移除旧的 `Lib` 文件夹再替换

上游在积极修复和推进pr的合并，<s>因此这个版本可能比较短寿。</s>

！！！由于 mpvnet 中止开发，这里也将会发布最后一版维护性质的简易包。

### 懒人包
其实没啥大变动。

### 简易包
无更新，沿用旧版。

<hr />
链接 <a href='https://github.com/hooke007/MPV_lazy/releases/tag/20230101'>mpv-lazy-20230101</a>

链接 <a href='https://github.com/hooke007/mpv.net_CM/releases/tag/6032'>mpvnet-CM-20221014</a>

**Comment by jiang-yuan** [comment]
补帧方案是以后都要自己去找或者自己配置吗....不够懒人了

**Reply by hooke007** [reply]
指什么自己找和自己配置？

**Reply by jiang-yuan** [reply]
![image](https://user-images.githubusercontent.com/36291537/210555369-c7a212bb-f13b-4253-bfea-e25b4caf242d.png)
vs补帧的配置啊

**Reply by hooke007** [reply]
rife自带集成，不需要什么配置。性能需求随着硬件升级逐渐不再是问题。

mvtools和svp的免费版多年未更新，自然淘汰。
手动配置不仅有完整的文档，而且有obs包存档在，复制几个文件后再改改input的事情

**Reply by jiang-yuan** [reply]
svp啥的1660ti对码率不高的4k还能补
rife的要求也太高了,4k电影完全没法补

**Reply by hooke007** [reply]
我现在发布在github的release上，svpflow肯定是不会再集成了，本身是商业软件做再分发不合适。mvtools的话，有足够的人反馈再说，目前来看需求不是很硬

**Reply by jiang-yuan** [reply]
rifeCuda装了这个dlc同样的N卡会跑的流畅一些吗

**Reply by hooke007** [reply]
1660没有张量核心，不属于20系

**Reply by csbxd** [reply]
mvtools不吃性能，还是挺有用的。

**Reply by hooke007** [reply]
mvtools不是不吃性能。。。我所有的配置都是以实用化（最低化性能开销和能看的质量平衡）为前提
意思是我尽量以我的最低标准提供最低性能需求的东西。如果不按照它来随便拉拉参数i7 8700k就直接跪了
![Snipaste_2023-01-07_03-56-34](https://user-images.githubusercontent.com/41094733/211130182-113d7f11-4717-4526-9d20-34615e54d984.jpg)

**Reply by wylsgame** [reply]
我的只能倍帧->24插帧到48帧, 如何像你一样补帧到60?

**Reply by hooke007** [reply]
除了2x默认60，打开文件自行检查用户选项

**Reply by wylsgame** [reply]
老大  修改rife_cuda.vpy 参数
FPS_num = 3   帧数倍数
GPU = 0
GPU_t = 6   显卡线程
WS_size = 6144   显卡显存
达到3倍帧数   应该这么改么?显卡占用有点高...
<img width="1181" alt="1111" src="https://user-images.githubusercontent.com/29670275/211185551-9310158c-6905-4c07-92d3-bf6fd09006a9.png">

**Reply by hooke007** [reply]
那就只改倍率，如果你不知道下面的选项是干啥的，那你不需要动它

gpu_t RTX4090这级别再考虑给到3
ws_size 不是你这么用的，不然初始值我不会给128，人专业卡都只需要1024

**Reply by wylsgame** [reply]
老大 我的显卡是3060ti  性能也挺差的  我现在只设置FPS_num = 3 其他改回默认值  显存占用降低,GPU占用还是85%  ..
        其实也不需要3倍70帧  60帧就挺好的  减少一点资源占用
        记得之前的版本我什么参数都不改 就可以2.5倍帧   不知道为何更新以后只能2倍帧?

**Reply by hooke007** [reply]
gpu占用低不了，因为渲染的帧数上去了
那你肯定是记错了，cuda从没支持过非整数。即使支持非整数，计算量可能比整数还大（2.5》3），因为要先算到3倍再抽帧或混成到2.5

**Reply by wylsgame** [reply]
大佬 我看你上面的图片里面 就是24帧原片补到60帧  是如何做到的?

**Reply by hooke007** [reply]
我回答的是 **mvtools**

**Reply by wylsgame** [reply]
mvtools 有没有配置说明?我想像你一样

**Reply by hooke007** [reply]
自行翻阅维基的链接

**Comment by wolfdaddyh** [comment]
Is there anyway you can implement this version (https://github.com/philipl/FFmpeg/tree/fruc-me) of ffmpeg to your mpv lazy package? It’s a pork that support nvfruc.

**Reply by hooke007** [reply]
You need upstream's building support. My version is the lite edition of shinchiro's build.
ref https://github.com/mpv-player/mpv/issues/11060

**Reply by wolfdaddyh** [reply]
Thank you. I’m sorry if this is not related but is there a guide anywhere for building mpv with ffmpeg support?

**Reply by hooke007** [reply]
Just check shinchiro's build page

**Comment by hooke007** [comment]
@taitaihoshi 我修正了FAQ中HDR直出的说明，vk下的自动切换有几率失败并完全破坏sdr的播放，不确定问题的来源是不是驱动，没有见过类似反馈。d3d11工作稳定，缺点是需要手动切换（但也只是一个系统自带快捷键的事情） https://github.com/hooke007/MPV_lazy/wiki/0_FAQ#%E8%A7%82%E7%9C%8Bhdr%E6%97%B6%E4%B8%8D%E5%81%9Asdr%E8%BD%AC%E6%8D%A2

**Reply by taitaihoshi** [reply]
在使用2023版之后无论是d3d11还是vulkan都已经可以正常直通hdr了，感谢更详细的说明

**Comment by Dashan-IZ** [comment]
![image](https://user-images.githubusercontent.com/78125220/211954946-5f81558e-71da-4b82-abd0-c50a66251e35.png)
4k屏幕太小选项，怎么让他变大，简直是看不清，希望增加webdav，这些选项

**Reply by hooke007** [reply]
https://github.com/hooke007/MPV_lazy/issues/229

**Reply by hooke007** [reply]
我不使用这协议，也没有原生实现
https://github.com/mpv-player/mpv/issues/11080
https://github.com/Kibakus/mpv-scripts

**Reply by Dashan-IZ** [reply]
我不太看得懂，我只会使用，如果能集成在主分支里边，最好不过了，wendav是个方便快捷功能

**Reply by Dashan-IZ** [reply]
选项太小这个怎么更改呢？？
需要修改那个配置文件和参数

**Reply by hooke007** [reply]
意思是官方没这功能，只有第三方写过它自己用的脚本。这个第三方表示它没有在windows上测试过是否可行。

**Reply by hooke007** [reply]
打开script-opts.conf拉到底的一个选项叫ui-scale

**Reply by Dashan-IZ** [reply]
谢谢，知道了，选项大小能解决吗

**Reply by hooke007** [reply]
大部分功能都可以通过浏览仓库的说明达成。
https://github.com/hooke007/MPV_lazy/blob/main/portable_config/script-opts.conf

当你不知道某个功能存不存在，首先应该尝试看对应的文档

**Reply by Dashan-IZ** [reply]
谢谢，修改script-opts-append = uosc-ui_scale=1.6可以完美解决，希望可以保留这个帖子的回复，他对我这个懒人很便捷，非常感谢

**Reply by Dashan-IZ** [reply]
![image](https://user-images.githubusercontent.com/78125220/211957640-64a96d13-cb26-45d1-9778-a30f30d7daeb.png)

**Reply by hooke007** [reply]
更新了FAQ的条目 https://github.com/hooke007/MPV_lazy/wiki/0_FAQ#%E9%AB%98%E5%88%86%E5%B1%8F%E7%9A%84%E7%95%8C%E9%9D%A2%E5%85%83%E7%B4%A0%E8%BF%87%E5%B0%8F

**Reply by dyphire** [reply]
uosc的ui_scale选项要根据显示器的实际dpi来设置，局限性的一点是如果有多个显示器并具有不同dpi的话它无法自适应，需要针对每个显示器重新设置ui_scale。更通用性的解决方法是启用mpv本身的hidpi-window-scale选项，其优劣性也很明显，本仓库已有相关讨论，不再累述。

至于webdav，在windows上完全可以直接挂载后使用，没有必要在mpv上单独实现。其他云盘类方案同理

**Comment by Dashan-IZ** [comment]
好的谢谢，但是通过其他webdav来实现会很臃肿，

dyphire ***@***.***> 于2023年1月12日周四 10:15写道：

**Reply by dyphire** [reply]
没有必要另开一楼。windows原生支持挂载webdav，不会的话建议善用搜索引擎；此外rclone挂载适用于绝大多数云盘/webdav方案（linux/windows通用）

**Reply by hooke007** [reply]
_他这个是在邮件回复的，邮件回复的会默认开新层。。。_

**Reply by zhongfly** [reply]
mpv自己专门实现一个webdav客户端的话，mpv作为一个播放器很臃肿…
在我看来mpv是自身集中于核心功能，但提供扩展性：通过脚本或者二次开发来完善各种各样的外围功能

**Comment by Cuttymon** [comment]
glsl-shaders怎么没有效果了

**Reply by Cuttymon** [reply]
我也不懂，排除了很久之后，发现注释掉 include = "~~/script-opts.conf" 就有效果了

**Reply by hooke007** [reply]
因为保存全局属性并恢复里有条目

**Reply by hooke007** [reply]
这在说明介绍的退出记住设置里就写了

**Reply by Cuttymon** [reply]
请问我要怎么在不注释的情况下让此选项生效

**Reply by hooke007** [reply]
重置保存的项目缓存
或
修改要保存的项目

**Comment by linger040831** [comment]
请问屏幕和显卡支持AMD FreeSync的情况下 是否还需要使用video-sync = display-resample 还是保持mpv默认就好？

**Reply by hooke007** [reply]
mpv暂不支持任何G-sync FreeSync AdaptiveSync技术。

**Comment by skiqing** [comment]
带带AMD玩吧。。

**Reply by hooke007** [reply]
我7950x可以超高速跑mvtools，这算不算（

**Reply by skiqing** [reply]
大佬我想说的是amd显卡有没有像rifecuda的组件？

**Reply by hooke007** [reply]
没什么人搞ROCm的东西，至于我们这种玩法属于小众里的小众的小众。建议有生之年早日转英伟达（

**Comment by n3sheltermercury** [comment]
窗口化暂停播放时屏幕不断闪烁，似乎是和gsync冲突，在控制面板已经配置固定刷新

**Reply by n3sheltermercury** [reply]
首选刷新率改为最高可用解决，建议在faq提醒一下

**Reply by hooke007** [reply]
看起来有点怪。安全起见三项全设置吧

**Comment by n3sheltermercury** [comment]
HDR在按照faq配置会过暗，把target-peak注释掉就正常了，也没有过曝，似乎是win11对亮度做了重新映射？使用的是vulkan方法

**Reply by n3sheltermercury** [reply]
注释掉是回去hdr转sdr了似乎。。。我的错，不过看起来很暗这个是为什么

**Reply by linger040831** [reply]
hdr转sdr就是会暗一点吧

**Reply by n3sheltermercury** [reply]
> hdr转sdr就是会暗一点吧

不是转sdr，是直接播放hdr

**Reply by hooke007** [reply]
win10下hdr源切换到sdr源会暴力过饱和，不清楚win11什么情况，所以目前只用d3d11

**Comment by taitaihoshi** [comment]
1.25倍速之后，某些视频的某些片段声音会失真，听起来像机器人在说话。其实半年前就发现了，但一直不知道怎么解决，用mpcbe倍速播放该视频同一个片段声音不会失真，怀疑是mpv的音调调整滤镜问题，不知道能否设置好。
The.Butterfly.Effect.2004.DC.1080p.BluRay.x265-RARBG.mp4  可以在rarbg上下载到这个视频，1:47:30秒开始，1.25倍速播放的话，对话声音是失真的。

**Reply by dyphire** [reply]
有可能是`video-sync=display-resample`的原因，该音视频同步模式下当音频以非常规速度播放时因为音频重采样后应用音调调整滤镜会导致明显的音高变化。

mpv最近新增了新的音视频同步模式`video-sync=display-tempo`来改善这一问题：https://github.com/mpv-player/mpv/commit/c4ec47a65e5840643024ba056ba4db20a4300ed2.
不过该音视频同步模式同样有使用限制：在常规音频速度下表现欠佳。建议配合条件配置来选择性地使用，可以参考：https://github.com/dyphire/mpv-config/commit/1ece4e78a74bc15dc6f12b5972f5b8241ac83972
```ini
[video-sync]
profile-desc=当音频调整速度超过阈值时改变音视频同步模式，以避免明显的音高变化
profile-cond=(speed * audio_speed_correction) > 0 and ((speed * audio_speed_correction) < 0.96 or (speed * audio_speed_correction) > 1.04)
profile-restore=copy-equal
video-sync=display-tempo
```

**Reply by hooke007** [reply]
非1倍速下默认插入的音频滤镜是定参，你可以手动修改参数尝试

**Comment by MrPinku** [comment]
更新了最新版本后，不会自动播放到文件夹里下一个文件，右键菜单查看播放列表也只有一个文件，是要自己设置什么吗？

而且我发现鼠标侧键绑定的上下文件也没效果，我看了看uosc的设置，自己改成了：

MBTN_BACK         script-binding uosc/prev-file
MBTN_FORWARD      script-binding uosc/next-file

这样就有效果了……但是我只会自己改快捷键，播放列表只有一个不能顺序播放的问题不知道怎么解决orz

**Reply by MrPinku** [reply]
补充一下，用的mpv-lazy-20230101.exe+mpv-lazy-20230101-hook.7z

**Reply by hooke007** [reply]
> 不会自动播放到文件夹里下一个文件

本来就不会

**Reply by hooke007** [reply]
自动加载功能自行查看脚本选项

**Reply by MrPinku** [reply]
噢，我从2021版更新过来的，原来默认不带，明白了。

**Reply by hooke007** [reply]
版本差异过大，我自己都不记得太久之前是什么样子了。旧脚本基本被剔除干净了

**Reply by hooke007** [reply]
。。。。我看到你在论坛的留言了

有没有可能，这个功能是内置的但是默认没开
https://github.com/hooke007/MPV_lazy/blob/3f68c5bd68aa7a7a79a212b72604ec6518a1c574/portable_config/script-opts.conf#L60

**Reply by MrPinku** [reply]
有没有可能，我真的不太懂。
哈哈哈，不过学到了，现在两种办法去开这个功能。
